scst-store/amr/data-model-and-concepts.hbs.md

```diff
@@ -0,0 +1,31 @@
+# AMR Data Model and Concepts
+
+This topic describes data models currently used in the Supply Chain Security Tools (SCST) – Artifact Metadata Repository (AMR).
+
+## <a id='overview'></a> Overview
+
+Here is a diagram that shows the data models currently used in AMR to store artifact information and the relation between them.
+
+![Diagram of the AMR Data Models](../images/AMR-data-model.jpg)
+
+## AMR Data Models
+
+Currently in the AMR there are two types of data models, `App` and `Location`.
+
+### Application
+
+The `App` data model represents the information of an application (i.e. replicaset) at a point in time.
+This information is sent to the AMR by `cloud events`, so each entry represents the status of the app when an event occurred.
```

Comment:
⛏️ Would changing `cloud events` to a link to a webpage that explains cloud events be a better choice? That way if readers don't know they could read up more on it?

Reply:
I'll link it to the cloud event doc under the amr folder